packages/vm/src/vm.ts

```diff
@@ -239,13 +239,19 @@ export class VM {
   /**
    * Returns a copy of the {@link VM} instance.
    *
-   * Note that the returned copy will share the same db as the original for the blockchain and the statemanager
+   * Note that the returned copy will share the same db as the original for the blockchain and the statemanager.
+   *
+   * Associated state and trie caches will be deleted and caches will be re-initialized for a more short-term
+   * focused usage, being less memory intense. To fine-tune this behavior (if the shallow copy has a longer life
+   * span e.g.) you can set the `downlevelCaches` option to `false`.
```

Comment:
```suggestion
   * Unless `downlevelCaches` is set to `false`, the state and trie caches will not be copied over to the new `vm` instance and will be re-initialized using the `ORDERED_MAP` 
   * cache type which is intended for a more short-term focused usage, being less memory intense. To have the original caches copied over (if the shallow copy has a longer life
   * span e.g.) you can set the `downlevelCaches` option to `false`.
```

Reply:
Feel free to ignore or modify this language but the original language feels a little unclear to me as to what we're actually doing with this param (at least for the uninitiated dev)

Reply:
Hmm, I had the argumentation a bit the other way around: I didn't want to go too much into the details because I would assume that someone dealing with the VM type is not aware about the cache datastructure type options over in the trie library. 🤔

I will do a hybrid and keep my sentence but add some details in brackets.